zerver/lib/bugdown/api_code_examples.py

```diff
@@ -138,8 +139,22 @@ def run(self, lines: List[str]) -> List[str]:
     def render_fixture(self, function: str, name: Optional[str]=None) -> List[str]:
         fixture = []
 
-        if name:
-            fixture_dict = zerver.lib.api_test_helpers.FIXTURES[function][name]
+        # We assume that if the function we're rendering starts with a slash
+        # it's a path in the endpoint and therefore it uses the new OpenAPI
+        # format.
+        if function.startswith('/'):
+            OPENAPI_SPEC_PATH = os.path.abspath(os.path.join(
+                os.path.dirname(__file__),
+                '../../../static/openapi/zulip.yaml'))
+
+            with open(OPENAPI_SPEC_PATH) as file:
+                yaml_parser = YamoleParser(file)
+                openapi = yaml_parser.data
+
+            path, method = function.rsplit(':', 1)
+            fixture_dict = (openapi['paths'][path][method]['responses'][name]
+                            ['content']['application/json']['schema']
+                            ['example'])
```

Comment:
Could this just call `load_openapi_fixtures`?  You might want to move it here and have the other call point (in tests) call it, but seems silly to have 2 copies of this code.

Reply:
(And possibly also `get_openapi_fixture`, too?)

Reply:
We could also put these in a `zerver/lib/openapi.py` if you'd prefer; might be a bit cleaner structurally.

Reply:
That's exactly what I felt was best, but I wasn't sure if creating a new file was appropiate. Will do it now, thanks!